e2e/test_color_theme.sh

```
@@ -0,0 +1,73 @@
+# test_color_theme.sh — E2E test for /color and /theme interaction.
+# Verifies: color targets, reset, theme switch, persistence.
+set -o errexit
+set -o nounset
+set -o pipefail
+if [[ "${TRACE-0}" == "1" ]]; then set -o xtrace; fi
+
+BINARY="${1:-./build/llama-cli}"
+export LLAMA_PROVIDER=mock
+export OLLAMA_HOSTS=""
+export OLLAMA_HOST=localhost
+export HOME="$(mktemp -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)